docs/rpc/openapi.yaml

```diff
@@ -551,3 +551,41 @@ paths:
             type: string
           description: The Stacks chain tip to query from. If tip == latest, the query will be run from the latest
             known tip (includes unconfirmed state).
+
+  /v2/block_proposal:
+    post:
+      summary: Validate a proposed Stacks block
+      tags:
+        - Mining
+      #operationId: ???
+      description: |
+        Used by miner to validate a proposed Stacks block.
+        Can accept either JSON or binary encoding.
+        
+        **This endpoint will only accept requests over the local loopback network interface.**
+      responses:
+        202:
+          description: Block proposal is valid
+        400:
+          description: Bad request
+        403:
+          description: Request not over loopback interface
+        429:
+          description: Too many requests
```

Comment:
Same here -- how many is too many, and what should the caller do if they hit this?

Reply:
I copied this from the previous PR, I don't think it makes sense to implement now, but when we implement async request handling we could add a max request queue depth that is set in the config file

Reply:
I think the answer here is "There is an ongoing request being evaluated. The `stacks-node` will only validate one block proposal at a time and will not queue such requests. The caller must wait until the ongoing request is completed."

Reply:
I think this isn't a reachable code until the async implementation is in place.